workflows/argo-events/workflowtemplates/update-nautobot-on-openstack-oslo-event.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: argoproj.io/v1alpha1
+metadata:
+  name: update-nautobot-on-openstack-oslo-event
```

Comment:
This whole file isn't needed cause we can use the generic one. This is just a copy and paste of the generic one with a new binary name.